src/EventExport/SortOrder.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\EventExport;
+
+use CultuurNet\UDB3\Deserializer\MissingValueException;
+
+final class SortOrder
+{
+    private string $property;
+    private string $order;
+
+    public function __construct(string $property, string $order)
+    {
+        $this->property = $property;
+        $this->order = $order;
+    }
+
+    public function getProperty(): string
+    {
+        return $this->property;
+    }
+
+    public function getOrder(): string
+    {
+        return $this->order;
+    }
+
+    /**
+     * @return array<string, string>
+     */
+    public function toArray(): array
+    {
+        return [
+            $this->getProperty() => $this->getOrder(),
+        ];
+    }
+
+    /**
+     * @param object|object[] $json
+     */
+    public static function fromJson($json): ?SortOrder
+    {
+        $hasProperty = isset($json->order->property);
+        $hasOrder = isset($json->order->order);
+
+        if ($hasProperty && !$hasOrder) {
```

Comment:
Very minor, as this will probably only be used by udb3-frontend, but most searches(e.g., SQL, ElasticSearch, ... have a default Order Mode(`asc`) While having to explicitly set `asc` or `desc` might be explicitly clear, It's also rather unique.

Reply:
Here you go 🙂 

[Apply asc as default sorting order if not provided](https://github.com/cultuurnet/udb3-backend/pull/1577/commits/3fcca09f45983218737a0285a9f151ae72cf9c13)